PathFind.cpp

```diff
@@ -1,106 +1,106 @@
-// Copyright 2024 Aditya Behrani
-#include <fstream>
-#include <iostream>
-#include <queue>
-#include <vector>
-#include "src/graph.cpp"
-#include "src/bfs.cpp"
-#include "src/dfs.cpp"
-#include "src/dijkstra.cpp"
-
-int START, GOAL;
-
-Graph* buildGraph(std::string filename) {
```

Comment:
you should almost never be using pointers in C++. This is a C pattern. Use references instead